packages/material-ui/src/Theme5/Mui5ContextProps.ts

```diff
@@ -1,69 +1,60 @@
-/** Use require for loading these libraries in case they are not available in order to perform a useful fallback
- */
-let mui = {};
-try {
-  mui = require('@mui/material');
-} catch (err) {
-  // purposely a no-op
-}
-
-// @ts-ignore What we are doing here isn't really good Typescript, but it works
-const {
-  Box,
-  Button,
-  Checkbox,
-  Divider,
-  FormControl,
-  FormControlLabel,
-  FormGroup,
-  FormHelperText,
-  FormLabel,
-  Grid,
-  IconButton,
-  InputLabel,
-  List,
-  ListItem,
-  ListItemIcon,
-  ListItemText,
-  MenuItem,
-  OutlinedInput,
-  Paper,
-  Radio,
-  RadioGroup,
-  Slider,
-  SvgIcon,
-  TextField,
-  Typography,
-} = mui;
+import { ComponentType } from 'react';
+import type {
```

Comment:
Notice that this is a *type* import, so it will be stripped out by the build (i.e. it won't ever make it into the bundle...it's purely for dev-time type-checking).